src/together/cli/api/endpoints.py

```diff
@@ -0,0 +1,346 @@
+@handle_api_errors
+def create(
+    client: Together,
+    model: str,
+    min_replicas: int,
+    max_replicas: int,
+    gpu: str,
+    gpu_count: int,
+    display_name: str | None,
+    no_prompt_cache: bool,
+    no_speculative_decoding: bool,
+    no_auto_start: bool,
+) -> None:
+    """Create a new dedicated inference endpoint."""
+    # Map GPU types to their full hardware ID names
+    gpu_map = {
+        "h100": "nvidia_h100_80gb_sxm",
+        "a100": "nvidia_a100_80gb_pcie" if gpu_count == 1 else "nvidia_a100_80gb_sxm",
+        "l40": "nvidia_l40",
+        "l40s": "nvidia_l40s",
+        "rtx-6000": "nvidia_rtx_6000_ada",
+    }
+
+    hardware_id = f"{gpu_count}x_{gpu_map[gpu]}"
+
+    response = client.endpoints.create(
+        model=model,
+        hardware=hardware_id,
+        min_replicas=min_replicas,
+        max_replicas=max_replicas,
+        display_name=display_name,
+        disable_prompt_cache=no_prompt_cache,
+        disable_speculative_decoding=no_speculative_decoding,
+        state="STOPPED" if no_auto_start else "STARTED",
+    )
+
+    # Print detailed information to stderr
+    click.echo("Created dedicated endpoint with:", err=True)
+    click.echo(f"  Model: {model}", err=True)
+    click.echo(f"  Min replicas: {min_replicas}", err=True)
+    click.echo(f"  Max replicas: {max_replicas}", err=True)
+    click.echo(f"  Hardware: {hardware_id}", err=True)
+    if display_name:
+        click.echo(f"  Display name: {display_name}", err=True)
+    if no_prompt_cache:
+        click.echo("  Prompt cache: disabled", err=True)
+    if no_speculative_decoding:
+        click.echo("  Speculative decoding: disabled", err=True)
+    if no_auto_start:
+        click.echo("  Auto-start: disabled", err=True)
+
+    click.echo("Endpoint created successfully, id: ", err=True)
+    # Print only the endpoint ID to stdout
+    click.echo(response.id)
+
+
+@endpoints.command()
+@click.argument("endpoint-id", required=True)
+@click.option("--json", is_flag=True, help="Print output in JSON format")
+@click.pass_obj
+@handle_api_errors
+def get(client: Together, endpoint_id: str, json: bool) -> None:
+    """Get a dedicated inference endpoint."""
+    endpoint = client.endpoints.get(endpoint_id)
+    print_endpoint(endpoint, json=json)
+
+
+@endpoints.command()
+@click.argument("endpoint-id", required=True)
+@click.pass_obj
+@handle_api_errors
+def stop(client: Together, endpoint_id: str) -> None:
+    """Stop a dedicated inference endpoint."""
+    client.endpoints.update(endpoint_id, state="STOPPED")
+    click.echo("Successfully stopped endpoint", err=True)
+    click.echo(endpoint_id)
+
+
+@endpoints.command()
+@click.argument("endpoint-id", required=True)
+@click.pass_obj
+@handle_api_errors
+def start(client: Together, endpoint_id: str) -> None:
+    """Start a dedicated inference endpoint."""
+    client.endpoints.update(endpoint_id, state="STARTED")
+    click.echo("Successfully started endpoint", err=True)
```

Comment:
Same as stoppped. In start it moves to `PENDING` and the `STARTING` and then `STARTED`. So we can change the message and maybe add a `--wait` option